### PR TITLE
Configurable package recipe export

### PR DESCRIPTION
**Pull request by santhonisz**
Changelog: Feature: Enable not exporting package recipe
Closes: #414 

- [x] Refer to the issue that supports this Pull Request.
- [x] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [x] I've read the [Contributing guide](https://github.com/conan-io/conan-package-tools/blob/develop/.github/CONTRIBUTING.md).
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008) style guides for Python code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -979,6 +979,20 @@ Using **CONAN_CLANG_VERSIONS** env variable in Travis ci or Appveyor:
 
     CONAN_CLANG_VERSIONS = "3.8,3.9,4.0"
 
+## Package Recipe Export
```
Comment (lasote) [inline README.md]: I would remove this section. It is already explained in the reference.

**Comment by lasote** [comment]
Please resolve also the conflicts.

**Comment by uilianries** [comment]
Done, let's wait for CI result.

**Comment by uilianries** [comment]
Fixed the conflicts, Let's see the CI result

**Pull request merged**